flink-rpc/flink-rpc-akka/src/test/java/org/apache/flink/runtime/rpc/akka/MessageSerializationTest.java

```diff
@@ -139,32 +139,6 @@ public void testSerializableRemoteMessageTransfer() throws Exception {
         assertThat(linkedBlockingQueue.take(), Is.<Object>is(expected));
     }
 
-    /**
-     * Tests that a message which exceeds the maximum frame size is detected and a corresponding
-     * exception is thrown.
-     */
-    @Test(expected = IOException.class)
-    public void testMaximumFramesizeRemoteMessageTransfer() throws Exception {
```

Comment:
Which tests now covers the case of oversized messages?

Reply:
Before this PR, Flink will eagerly check the oversized messages. Now, when there is an oversized message, the job will fail with Akka's `AskTimeoutException` and there will be an error log from Akka which indicates the exact size of that message. As Flink does not take the responsibility to detect oversized messages, I think we don't need such tests anymore.

Reply:
There should still be a test that checks the behavior if this error case occurs. E.g., that the response future is completed exceptionally.